tests/scripts/actionutils.sh

```diff
@@ -639,7 +644,7 @@ function wait_for_osds() {
     if [[ $res -lt $expect ]] ; then
         echo "Never reached ${expect} OSDs"
         return -1
-    fi    
```

Comment:
I appreciate such fixes but ideally they should be carried out in separate PRs.

Reply:
Thanks, that's me being lazy. Will definitely do next time

Reply:
good editor 😃